list_of_subprojects.md

```diff
@@ -6,7 +6,7 @@ At the highest level, any Github repository under any official Jupyter GitHub or
 
 The following Jupyter Subprojects have their own formal Subproject Council that elects and maintains an SSC representative:
 
-- [JupyterLab](https://github.com/jupyterlab/team-compass)
+- [Jupyter Frontends](https://github.com/jupyterlab/team-compass) and [additional link](https://github.com/jupyter/notebook-team-compass)
```

Comment:
Which team-compass would be the final state? I don't think we should list two here; rather pick one where the "Frontend" Teams live.

Reply:
I would favor using jupyterlab/team-compass as it is in a separate GitHub organization which aligns better with the other subprojects.